package.json

```diff
@@ -19,7 +19,7 @@
     "@rsksmart/powpeg-redeemscript-parser": "^1.0.0",
     "bn.js": "^4.11.8",
     "bridge-transaction-parser-hop400": "github:rsksmart/bridge-transaction-parser#v0.4.0-beta",
-    "bridge-transaction-parser-fingerroot500": "github:rsksmart/bridge-transaction-parser#v0.5.0-beta",
+    "bridge-transaction-parser": "github:rsksmart/bridge-transaction-parser#v0.5.0-beta",
```

Comment:
why version 0.5? Shouldn't it be the latest one?

Reply:
Updated.